Source/ACE.Server/WorldObjects/WorldObject_Properties.cs

```diff
@@ -2515,6 +2515,12 @@ public int? PortalReqMaxValue
             set { if (!value.HasValue) RemoveProperty(PropertyInt.PortalReqMaxValue); else SetProperty(PropertyInt.PortalReqMaxValue, value.Value); }
         }
 
+        public uint? CreatedByAccountId
+        {
+            get => GetProperty(PropertyDataId.CreatedByAccountId);
+            set { if (!value.HasValue) RemoveProperty(PropertyDataId.CreatedByAccountId); else SetProperty(PropertyDataId.CreatedByAccountId, value.Value); }
```

Comment:
could we modify the set to only work if it's null? that way it's effectively only set once and immune to /setproperty and other shenanigans

Reply:
Turns out setproperty has its own code path to update the value, so I did make this change, but also had to update setproperty directly.